### PR TITLE
added theme Simplistick

### PR DESCRIPTION
**Pull request by renegat59**
Simple theme with prompt in the new line below the pwd

### CODE REVIEW

File: themes/simplistick/simplistick.theme.bash [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+LAST_PROMPT=""
+function prompt_command() {
+    local prompt="${green}%"
+    local new_PS1="${white}>$(scm_char) $(scm_prompt_info)$(ruby_version_prompt)${bold_cyan}${green}\w"
+    local wrap_char="\n"
+    PS1="${new_PS1}${wrap_char}${prompt}${reset_color} "
+}
+
+safe_append_prompt_command prompt_command
```
Comment (jpmcb) [inline themes/simplistick/simplistick.theme.bash]: For consistency and best practice, there should be a new line at the end of the file

File: themes/simplistick/simplistick.theme.bash [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+VIRTUALENV_THEME_PROMPT_SUFFIX=") "
+
+RBENV_THEME_PROMPT_PREFIX="("
+RBENV_THEME_PROMPT_SUFFIX=") "
+
+RBFU_THEME_PROMPT_PREFIX="("
+RBFU_THEME_PROMPT_SUFFIX=") "
+
+function rvm_version_prompt {
+  if which rvm &> /dev/null; then
```
Comment (jpmcb) [inline themes/simplistick/simplistick.theme.bash]: Which is non-standard across unix systems, this should be:
```
if command -v rvm
```

File: themes/simplistick/simplistick.theme.bash [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    [ "$rvm_current" !=  "$rvm_default" ] && ( echo -e "$RVM_THEME_PROMPT_PREFIX$rvm_current$RVM_THEME_PROMPT_SUFFIX" )
+  fi
+}
+
+function git_prompt_info {
+  git_prompt_vars
+  echo -e "$SCM_PREFIX${yellow}$SCM_BRANCH$SCM_STATE$SCM_GIT_AHEAD$SCM_GIT_BEHIND$SCM_GIT_STASH$SCM_SUFFIX"
+}
+
+LAST_PROMPT=""
```
Comment (jpmcb) [inline themes/simplistick/simplistick.theme.bash]: This appears to be unused and the only other reference is locally in `iterate.theme.bash`. This appears to be unneeded.

File: themes/simplistick/simplistick.theme.bash (hunk 2 above)
Comment (nwinkler) [reply themes/simplistick/simplistick.theme.bash]: Bash-it provides the `_command_exists` function for this purpose:

```bash
if _command_exists rvm
```

**Comment by nwinkler** [comment]
Fully agree with @jpmcb - I'd rather have more customization in the existing themes, then yet another theme that modifies one or two lines from another theme...

**Comment by NoahGorny** [comment]
Hey, seems like this PR has gone stale. So I am closing it. Thanks for contributing!

**Pull request closed without merging**